gateway/openapi.yaml

```diff
@@ -0,0 +1,327 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Supplementary Data Service (SDS)
+  description: API gateway for SDS
+schemes:
+  - https
+securityDefinitions:
+  UserSecurity:
+    type: basic
+security:
+  - UserSecurity: []
+paths:
+  /v1/unit_data:
+    post:
```

Comment:
I think the method should be get at the unit data endpoint

Reply:
Nice spot will change